charts/victoria-metrics-k8s-stack/values.yaml

```diff
@@ -716,6 +716,31 @@ vmalert:
     #    hosts:
     #      - vmalert.domain.com
 
+vmauth:
+  enabled: false
+  annotations: {}
+  # -- (object) full spec for VMAuth CRD. Allowed values described [here](https://docs.victoriametrics.com/operator/api#vmauthspec)
+  spec:
+    discover_backend_ips: true
```

Comment:
vmauth spec doesn't have such setting at spec https://docs.victoriametrics.com/operator/api/

Reply:
it does https://docs.victoriametrics.com/operator/api/#vmauthspec